File: Projeto_2/projeto_2 - Kaella Sales/src/Cliente.cpp
#include "../include/Cliente.h"
#include <iostream>


Cliente::Cliente(std::string n, std::string cepef){
nome = n;
cpf = cepef;
}

void Cliente::ExibirInformacoes(){
    std::cout << "\n" << std::endl;
    std::cout << "Cliente" << std::endl;
    std::cout << "Nome: " << nome << std::endl;
    std::cout << "CPF: " << cpf << "\n" << std::endl;
}

File: Projeto_2/Projeto_2 - Alan Lima/src/cliente.py
class Cliente:
    def __init__(self, nome: str, cpf: str):
        self.__nome = nome
        self.__cpf = cpf

    def exibir_informacoes(self):
        informacoes = (
            f"Informações do Cliente:\n"
            f"Nome: {self.__nome}\n"
            f"cpf: {self.__cpf}\n"
        )
        return informacoes

    def __repr__(self):
        return f"Cliente(nome='{self.__nome}', cpf='{self.__cpf}')"
